client/package-lock.json

```diff
@@ -0,0 +1,16910 @@
+{
```

Comment:
can we ignore this file? It conflicts having `yarn.lock` and `package-lock.json`